From de8af64dad24b35599617dc887e5bccc5397985c Mon Sep 17 00:00:00 2001
From: Robert Bogart <Robert.bogart@joyent.com>
Date: Mon, 24 Sep 2018 20:48:28 +0000
Subject: [PATCH] MANTA-3854 Mako sometimes fails to create manifests due to
 limitations in 'find' Reviewed by: Robert Mustacchi <rm@joyent.com> Reviewed
 by: Trent Mick <trent.mick@joyent.com> Approved by: Trent Mick
 <trent.mick@joyent.com>

---
 targets.json.in | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/targets.json.in b/targets.json.in
index a39048f..b6a0f78 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1223,10 +1223,12 @@ cat <<EOF
     "build_platform": "20151126T062538Z",
     "image_name": "manta-storage",
     "image_description": "Manta storage",
-    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
-    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
+    "// image_uuid": "triton-origin-multiarch-18.1.0@1.0.1",
+    "image_uuid": "b6ea7cb4-6b90-48c0-99e7-1d34c2895248",
     "pkgsrc": [
-      "pcre-8.38"
+      "pcre-8.42",
+      "findutils-4.6.0nb1",
+      "gawk-4.1.4nb1"
     ],
     "repos": [
       {"url": "git@github.com:joyent/manta-mako.git"}
-- 
2.21.0

